core/src/main/scala/org/apache/spark/scheduler/cluster/YarnSchedulerBackend.scala

```diff
@@ -17,6 +17,10 @@
 
 package org.apache.spark.scheduler.cluster
 
+import java.util.regex.Pattern
+
+import org.apache.hadoop.yarn.api.records.ContainerExitStatus
```

Comment:
Good catch, I updated some code and forgot to take these out. I'll make sure, other places are kosher too.